app/assets/javascripts/directives/task_item.js

```diff
@@ -41,6 +41,17 @@ angular.module("OpenHq").directive("taskItem", function($rootScope, Task, Confir
           $scope.task.update();
         }
       });
+
+      $(".tasks ul.sortable").sortable({
+        items: "li",
+        update: function() {
+          var $this = $(this),
+              story_id = $this.attr('data-story-id'),
```

Comment:
you should be able to get the story slug from route params or the ID from $scope